docs/tutorial/security.md

```diff
@@ -370,8 +370,7 @@ session.defaultSession.webRequest.onHeadersReceived((details, callback) => {
 
 ### CSP Meta Tag
 
-CSP's preferred delivery mechanism is an HTTP header. It can be useful, however,
-to set a policy on a page directly in the markup using a `<meta>` tag:
+CSP's preferred delivery mechanism is an HTTP header, however it is not possible to use this method when loading a resource using the `file://` protocol. It can be useful in some cases, such as using the `file://` protocol, to set a policy on a page directly in the markup using a `<meta>` tag:
 
```

Comment:
good stuff :clap:

i think it's worth also noting a few other things:

1. the example usage of `onHeadersReceived()` is wrong - specifically the `{ responseHeaders: `default-src 'none'` }` format should be 👉 https://stackoverflow.com/a/52243138/579167 (i think?)
2. CSP headers only need to be set for `Content-Type: text/html` (and `application/xhtml+xml` and `application/xml` - anything that generates an interactive document) responses

Reply:
@Slapbox agree with the above; if you could address these comments we can look to getting this merged!

Reply:
@codebytere @busticated thanks for your reviews. I'd be happy to update this, but because I've never actually set headers this way due to exclusive use of the `file://` protocol in my projects, I'd be worried about updating the docs with inaccurate information. Can anyone confirm the below is accurate?

> the example usage of onHeadersReceived() is wrong - specifically the { responseHeaders:default-src 'none'} format should be point_right https://stackoverflow.com/a/52243138/579167 (i think?)

Reply:
The other notes are additional to other chapters, and do not conflict with this PR which only adds new content to "CSP Meta Tag", please start a new PR to add new content.